src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -0,0 +1,348 @@
+import * as MicrosoftGraph from '@microsoft/microsoft-graph-types';
+import { customElement, html, property } from 'lit-element';
+import { Providers } from '../../Providers';
+import { ProviderState } from '../../providers/IProvider';
+import { getEmailFromGraphEntity } from '../../utils/graphHelpers';
+import * as svgHelper from '../../utils/svgHelper';
+import { PersonCardInteraction } from '../mgt-person/mgt-person';
+import { MgtTemplatedComponent } from '../templatedComponent';
+import { styles } from './mgt-person-card-css';
+
+/**
+ * Web Component used to show detailed data for a person in the
+ * Microsoft Graph
+ *
+ * @export
+ * @class MgtPersonCard
+ * @extends {MgtTemplatedComponent}
+ */
+@customElement('mgt-person-card')
+export class MgtPersonCard extends MgtTemplatedComponent {
+  /**
+   * Array of styles to apply to the element. The styles should be defined
+   * using the `css` tag function.
+   */
+  static get styles() {
+    return styles;
+  }
+
+  /**
+   * Set the person details to render
+   *
+   * @type {(MicrosoftGraph.User
+   *     | MicrosoftGraph.Person
+   *     | MicrosoftGraph.Contact)}
+   * @memberof MgtPersonCard
+   */
+  @property({
+    attribute: 'person-details',
+    type: Object
+  })
+  public personDetails: MicrosoftGraph.User | MicrosoftGraph.Person | MicrosoftGraph.Contact;
+  /**
+   * Set the image of the person
+   * Set to '@' to look up image from the graph
+   *
+   * @type {string}
+   * @memberof MgtPersonCard
+   */
+  @property({
+    attribute: 'person-image',
+    type: String
+  })
+  public personImage: string;
+
+  /**
+   * Gets or sets whether additional details section is rendered
+   *
+   * @type {string}
+   * @memberof MgtPersonCard
+   */
+  @property({
+    attribute: 'is-extended',
+    type: Boolean
+  })
+  public isExtended: boolean = false;
```

Comment:
While using this property, it makes more sense to call it isExpanded - thoughts?

Reply:
Makes sense to me and avoids a potential conflict with:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/extends

Reply:
updates to Person and Person-card:
a50cef2